actions/workflows/dns_record_claim.yaml

```diff
@@ -97,6 +109,20 @@ tasks:
         do:
           - get_dns_zone
 
+  # Verify the IP address is available for use
+  test_ip_address:
+    action: menandmice.test_ip_address
+    input:
+      server: "{{ ctx().server }}"
+      session: "{{ ctx().session }}"
+      transport: "{{ ctx().transport }}"
+      range_name: "{{ ctx().range_name }}"
+      ip_address: "{{ ctx().ip_address }}"
+    next:
```

Comment:
what happens if this fails?

Reply:
the workflow fails. Nothing has been claimed yet so no remediation is needed